Lib/test/support/threading_helper.py

```diff
@@ -3,11 +3,27 @@
 import functools
 import sys
 import threading
+from socket import socket
 import time
 import unittest
+from concurrent.futures import ThreadPoolExecutor
+from test.support.socket_helper import bind_port, HOST
 
 from test import support
 
+_thread_pool = None
+
+
+def _release():
+    global _thread_pool
+    _thread_pool = None
+
+
+def init():
+    global _thread_pool
+    _thread_pool = ThreadPoolExecutor()
+    unittest.addModuleCleanup(_release)
```

Comment:
I've added `init()` to avoid a `modified environment` test failure due to a leaked object. If there's a better solution, I'll use that instead.